packages/dd-trace/src/config.js

```diff
@@ -8,8 +8,6 @@ const log = require('./log')
 const pkg = require('./pkg')
 const coalesce = require('koalas')
 const tagger = require('./tagger')
-const get = require('../../datadog-core/src/utils/src/get')
-const has = require('../../datadog-core/src/utils/src/has')
```

Comment:
I just realized that this file is the only one that uses `get`, `has` and `set`, so these two files and their tests should actually be deleted as part fo this PR